js/sizer.js

```diff
@@ -361,7 +322,7 @@ Sum of the size of the Keys + Sum of the size of the static columns + Number of
 
 //Ugly....
 function downloadYaml(filename) {
-  var text = $("#tableDef").val();
+  var text = editor.getValue();
```

Comment:
I'm not sure if we would use the variable (processed version of the table) for the file output, or if we should use the table def verbatim as the user created it. I think we're better off using processed version to ensure that we have a valid yaml.